### ISSUE TITLE
[make -jx] fails

### ISSUE THREAD

**Issue by komh** [komh/make-os2#1]
For example, ```make -j2``` fails like this:

```
make: *** cannot open jobserver P:\tmp/GMfifo1943: No such file or directory.  Stop.

```

**Comment by komh** [comment]
Try this: 
[make_test.zip](https://github.com/komh/make-os2/files/13487549/make_test.zip)

**Comment by komh** [comment]
Fixed by commit 2c21e4421e73b060a557b87061d2c302e4503a0d.
